src/test/resources/application.properties

```diff
@@ -24,3 +24,11 @@ activiti.keycloak.client-password=client
 # this user needs to have the realm management roles assigned
 # one option to deal with password being here would be spring-boot-jasypt, keycloak may support others (can it be set as env variable?)
 
+spring.audit.liquibase.change-log=classpath:config/audit/liquibase/master.xml
+spring.audit.liquibase.database-change-log-table=DATABASECHANGELOG_AUDIT
+spring.audit.liquibase.database-change-log-lock-table=DATABASECHANGELOGLOCK_AUDIT
+
+spring.query.liquibase.change-log=classpath:config/query/liquibase/master.xml
+spring.query.liquibase.database-change-log-table=DATABASECHANGELOG_QUERY
+spring.query.liquibase.database-change-log-lock-table=DATABASECHANGELOGLOCK_QUERY
+
```

Comment:
if you already touch this would be nice to turn banner off for tests: `spring.main.banner-mode=off`